app/src/main/java/org/oppia/app/player/exploration/ExplorationActivity.kt

```diff
@@ -21,9 +22,11 @@ private const val TAG_STOP_EXPLORATION_DIALOG = "STOP_EXPLORATION_DIALOG"
 const val TAG_HINTS_AND_SOLUTION_DIALOG = "HINTS_AND_SOLUTION_DIALOG"
 
 /** The starting point for exploration. */
-class ExplorationActivity : InjectableAppCompatActivity(), StopExplorationInterface, StateKeyboardButtonListener,
-  AudioButtonListener, HintsAndSolutionListener, RouteToHintsAndSolutionListener, RevealHintListener,
-  RevealSolutionInterface {
+class ExplorationActivity : InjectableAppCompatActivity(), StopExplorationInterface,
+  StateKeyboardButtonListener,
+  AudioButtonListener, HintsAndSolutionListener, RouteToHintsAndSolutionListener,
+  RevealHintListener,
+  RevealSolutionInterface, DeafultFontSizeStateListener {
```

Comment:
To make it more readable follow one of the below approaches:
```
class ExplorationActivity : InjectableAppCompatActivity(), StopExplorationInterface,
  StateKeyboardButtonListener, AudioButtonListener, HintsAndSolutionListener,
  RouteToHintsAndSolutionListener, RevealHintListener, RevealSolutionInterface,
  DeafultFontSizeStateListener {
```
class ExplorationActivity : InjectableAppCompatActivity(),
  StopExplorationInterface,
  StateKeyboardButtonListener,
  AudioButtonListener,
  HintsAndSolutionListener,
  RouteToHintsAndSolutionListener,
  RevealHintListener,
  RevealSolutionInterface,
  DeafultFontSizeStateListener {
```

```

Reply:
Updated alignment for better readability.

Reply:
+1 this is not updated yet.

Reply:
+1

Reply:
Done.